official/resnet/resnet_run_loop.py

```diff
@@ -471,3 +472,11 @@ def define_resnet_flags(resnet_size_choices=None):
     flags.DEFINE_string(**choice_kwargs)
   else:
     flags.DEFINE_enum(enum_values=resnet_size_choices, **choice_kwargs)
+
+  # The current implementation of ResNet v1 is numerically unstable when run
+  # with fp16 and will produce NaN errors soon after training begins.
+  msg = 'ResNet version 1 does not currently support fp16.'
```

Comment:
'ResNet version 1 is not currently supported with fp16. Please use version 2 instead.'